core/templates/dev/head/pages/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -63,32 +65,25 @@ oppia.factory('ThreadDataService', [
           target_id: _expId
         };
       }
-      var sPromise = $http.get(_SUGGESTION_LIST_HANDLER_URL, {
+      var suggestionsPromise = $http.get(_SUGGESTION_LIST_HANDLER_URL, {
         params: params
       });
 
-      $q.all([fPromise, sPromise]).then(function(res) {
+      $q.all([threadsPromise, suggestionsPromise]).then(function(res) {
         _data.feedbackThreads = res[0].data.threads;
         _data.suggestionThreads = [];
         if (constants.USE_NEW_SUGGESTION_FRAMEWORK) {
+          var suggestionThreads = res[0].data.suggestion_thread_dicts;
```

Comment:
Any checks to ensure that length of suggestionThreads is same as length of suggestions?

If not same then should $log.error to backend and handle that case appropriately in frontend.

Reply:
Oops.. missed this comment.